config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    dba:"postgres:localhost:5432/bundle",
+    DARKSKY_API_KEY:"aa9770871dcf844f5ca90bbdcd087e5b"
```

Comment:
Just checking: these are intentionally checked in?

Reply:
that is not supposed to be there, thanks for catching it